Dockerfile

```diff
@@ -34,7 +34,7 @@
 # /////////////////////////////////////////////////////////////////////////////
 
 FROM usdotfhwastol/autoware.ai:carma-system-3.6.0 AS base-image
-FROM usdotfhwastol/autoware.ai:zephyr AS base-image
+# FROM usdotfhwastol/autoware.ai:zephyr AS base-image
```

Comment:
Its really better to just delete this line instead of commenting it out to avoid confusion